pages/linux/upnpc.md

```diff
@@ -0,0 +1,20 @@
+# upnpc
+
+> Configure port forwarding rules on your router via the UPnP protocol.
+> More information: <http://miniupnp.free.fr/>.
+
+- Forward external TCP port 80 to port 8080 on a local machine:
+
+`upnpc -a {{xxx.xxx.xxx.xxx}} 8080 80 tcp`
```

Comment:
```suggestion
`upnpc -a {{external_ip_address}} 8080 80 tcp`
```
Optionally, we can update the placeholder to something verbose (to align with other similar pages), but the previous version works too.

Reply:
I don't like the `external_ip_address` placeholder. I think it would be better to use an IP address instead. For example:
```suggestion
`upnpc -a {{192.168.0.1}} 8080 80 tcp`
``` 
(this one is used in pages/windows/netsh-interface-portproxy.md)
or based on [RFC5737](https://www.rfc-editor.org/rfc/rfc5737):
```suggestion
`upnpc -a {{203.0.113.254}} 8080 80 tcp`
```
```suggestion
`upnpc -a {{192.0.2.10}} 8080 80 tcp`
```
or any other address